pandas/io/w1902.py

```diff
@@ -0,0 +1,67 @@
+import sys
```

Comment:
What do these two files do?

Reply:
Sorry, It was added by mistake! I could remove it right away.

Reply:
Should I make another pull request after resolving this?